src/gobexport/test.py

```diff
@@ -114,34 +114,37 @@ def test(catalogue):
 
             for filename in filenames:
                 # Check the previously exported file at its temporary location
-                obj_info, obj = _get_file(conn_info, f"{EXPORT_DIR}/{catalogue}/{filename}")
-
-                # Clone check so that changes to the check file don't affect other runs
-                check = copy.deepcopy(_get_check(checks, filename))
-
-                # Report results with the name of the matched file
-                matched_filename = obj_info['name'] if obj_info else filename
+                obj_info, obj_contents = _get_file(conn_info, f"{EXPORT_DIR}/{catalogue}/{filename}")
```

Comment:
No functional changes in this file. Just some renaming of variables and structural changes to make the core more readable and understandable.

These changes are in a separate commit, so easy to hide. See PR description